_includes/videowrapper.html

```diff
@@ -2,4 +2,5 @@
 
 <video width="100%" height="auto" controls>
     <source src="{{ include.url }}" type="video/mp4">
-</video>
\ No newline at end of file
+    <track src="{{ include.subs }}" kind="subtitles" srclang="en" label="English">
```

Comment:
```suggestion
    <track src="{{ site.baseurl }}/{{ include.subs }}" kind="subtitles" srclang="en" label="English">
```

Reply:
Adding this here removes the need to provide the full path for each video AND it lets us test it locally beforehand